app/javascript/application.js

```diff
@@ -1,16 +1,16 @@
 // Entry point for the build script in your package.json
 
-import "jquery"; // Blacklight 7 needs jQuery. Remove when we upgrade to Blacklight 8?
-import bootstrap from "bootstrap/dist/js/bootstrap";
-global.bootstrap = bootstrap; // Required for Blacklight 7 so it can manage the modals
+import 'jquery' // Blacklight 7 needs jQuery. Remove when we upgrade to Blacklight 8?
```

Comment:
JSLint prefers double quotes.  Why did you change all of these to single?

Reply:
Probably Niqui's editor did this without her noticing.  I changed it back.

Reply:
Huh, maybe the preference changed in newer versions of jslint?